changelog/april2024/2024-04-03-block-storage-added-low-latency-offers-integration-in-c.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Low Latency offers integration in Cockpit
+status: added
+author:
+    fullname: 'Join the #block-storage channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-04-03
+category: storage
+product: block-storage
+---
+
+A new dashboard dedicated to block storage Low Latency offers is now available.
+Your Low Latency block storage consumption is now available for each AZ in order to improve the management of the capacity you consum.
+Performance information are still not available, we are still working on it.
```

Comment:
```suggestion
We are currently in the process of working on displaying performance information, and it will be made available as soon as the feature ready.
```